README.md

```diff
@@ -142,6 +142,10 @@ Devbox currently detects the following languages:
 - Go
 - Python (Poetry)
 
+Shell only support:
```

Comment:
Technically we let you create a shell for any language, you just need to declare the packages you need in `devbox.json`. So I'm slightly worried that by phrasing these as "shell-only" people will interpret it as, languages *not* in this list are not supported at all.

Thoughts?